pjmedia/src/pjmedia/rtp.c

```diff
@@ -281,6 +281,7 @@ PJ_DEF(void) pjmedia_rtp_session_update2( pjmedia_rtp_session *ses,
     if (p_seq_st) {
 	p_seq_st->status.value = seq_st.status.value;
 	p_seq_st->diff = seq_st.diff;
+	p_seq_st->status.flag.badssrc = seq_st.status.flag.badssrc;
```

Comment:
Copying `status.value` above should be sufficient to copy all flags including `badssrc`, shouldn't it?